src/ape_test/provider.py

```diff
@@ -58,6 +58,9 @@ def disconnect(self):
     def update_settings(self, new_settings: dict):
         pass
 
+    def unlock_account(self, address: AddressType) -> bool:
```

Comment:
This is pretty cool; I'd like to see some new tests showcasing this.

Reply:
will add 1-2 tests

Reply:
Oh wait I just realized this was a thing: https://github.com/ethereum/eth-tester/blob/master/eth_tester/main.py#L205

Can we use their official method?

Reply:
:s i just tried this and it didnt workout well, so the problem is you can create a new account using private key and lock/unlock to that account, so you cant random add an account with an address, so i will revert this because eth-tester doesnt support this.

Reply:
Thanks for trying!